sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -1962,3 +1962,27 @@ ALTER TABLE `immunizations` ADD `uuid` binary(16) DEFAULT NULL;
 CREATE UNIQUE INDEX `uuid` ON `immunizations` (`uuid`);
 #EndIf
 
+#IfMissingColumn lists uuid
+ALTER TABLE `lists` ADD `uuid` binary(16) DEFAULT NULL;
+#EndIf
+
+#IfUuidNeedUpdate lists
+#EndIf
+
+#IfNotIndex lists uuid
+CREATE UNIQUE INDEX `uuid` ON `lists` (`uuid`);
+#EndIf
+
+#IfMissingColumn lists verification
+ALTER TABLE `lists` ADD `verification` VARCHAR(36) NOT NULL DEFAULT "unconfirmed";
```

Comment:
not sure if making all allergies unconfirmed makes sense for upgraders. would probably:
1. make blank by default (so then upgraders entries will be blank)
2. if blank, then fhir should return unconfirmed for the resource

Reply:
Ooops, Sometimes I just be selfish and think from FHIR perspective and forget there is also an web app :sweat_smile: